torchsde/_core/misc.py

```diff
@@ -98,3 +98,12 @@ def flat_to_shape(tensor, shapes, length=()):
         tensor_list.append(tensor[..., total:next_total].view((*length, *shape)))
         total = next_total
     return tensor_list
+
+
+def _stable_division(x, y, epsilon=1e-7):
+    y = torch.where(
+        y.abs() > epsilon,
+        y,
+        torch.ones_like(y).fill_(epsilon) * y.sign()
```

Comment:
Quick comment: can this be `torch.full_like(y, epsilon * y.sign())` instead?
(Untested, might have the syntax slightly wrong.)

Reply:
Sure!
APi says that second arg should be a number. This works for me: `torch.full_like(y, epsilon) * y.sign()`

Reply:
Done!